pkg/kube/apisix/apis/config/v2beta1/types.go

```diff
@@ -197,3 +197,35 @@ type ApisixRouteList struct {
 	metav1.ListMeta `json:"metadata" yaml:"metadata"`
 	Items           []ApisixRoute `json:"items,omitempty" yaml:"items,omitempty"`
 }
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:subresource:status
+
+// ApisixPluginConfig is the Schema for the ApisixPluginConfig resource.
+// An ApisixPluginConfig is used to support a group of plugin configs
+type ApisixPluginConfig struct {
+	metav1.TypeMeta   `json:",inline" yaml:",inline"`
+	metav1.ObjectMeta `json:"metadata" yaml:"metadata"`
+
+	// Spec defines the desired state of ApisixPluginConfigSpec.
+	Spec   ApisixPluginConfigSpec `json:"spec" yaml:"spec"`
+	Status ApisixStatus           `json:"status,omitempty" yaml:"status,omitempty"`
+}
+
+// ApisixPluginConfigSpec defines the desired state of ApisixPluginConfigSpec.
+type ApisixPluginConfigSpec struct {
+	Desc string `json:"desc,omitempty" yaml:"desc,omitempty"`
+	// Plugins contains a list of ApisixRouteHTTPPluginConfig
+	Plugins []ApisixRouteHTTPPluginConfig `json:"plugins,omitempty" yaml:"plugins,omitempty"`
+}
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ApisixPluginConfigList contains a list of ApisixPluginConfig.
+type ApisixPluginConfigList struct {
```

Comment:
It can be deleted, no need for it.

Reply:
do u meaning the `ApisixPluginConfigList`?
ok, i will delete it.

Reply:
yes

Reply:
Is this no longer used? Or how do we list ApisixPluginConfig?